### PR TITLE
Elasticsearch documentation

### PR DESCRIPTION
**Pull request by meyerbaptiste**

### CODE REVIEW

File: core/data-providers.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Data Providers
 
 To retrieve data exposed by the API, API Platform uses classes called **data providers**. A data provider using [Doctrine
-ORM](http://www.doctrine-project.org/projects/orm.html) to retrieve data from a database is included with the library and
-is enabled by default. This data provider natively supports paged collections and filters. It can be used as is and fits
-perfectly with common usages.
+ORM](http://www.doctrine-project.org/projects/orm.html) to retrieve data from a database and a data provider using
```
Comment (dunglas) [inline core/data-providers.md]: could we mention Mongo too?

File: core/filters.md [hunk 2]
```diff
@@ -563,6 +563,141 @@ It means that the filter will be **silently** ignored if the property:
...
+ */
+class User
+{
+    // ...
+}
+```
+
+Given that the collection endpoint is `/users`, you can filter users by gender and age.
+
+`/users?gender=female` will return all users whose gender is `female`.
```
Comment (dunglas) [inline core/filters.md]: It could be nice to add an example of how to do a full text search.

Comment (dunglas) [reply core/filters.md]: Adding an example of how to use boosts (if supported) would be nice too

Comment (meyerbaptiste) [reply core/filters.md]: Match filter added. Boosts are not yet supported.

**Comment by dunglas** [comment]
Thanks @meyerbaptiste

**Pull request merged**